region/wreckedship/main/Attic.json

```diff
@@ -214,6 +214,210 @@
       },
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "Leave Spinning",
+      "requires": [],
+      "exitCondition": {
+        "leaveSpinning": {
+          "remoteRunway": {
+            "length": 10,
+            "openEnd": 1
+          },
+          "minExtraRunSpeed": "$0.8"
+        }
+      }
+    },
+    {
+      "link": [1, 1],
+      "name": "Leave Spinning (Space Jump)",
+      "requires": [
+        "SpaceJump"
```

Comment:
I've been thinking about Attic for months. It needs a major overhaul to model the enemies, but if you can kill Phantoon, the enemies shouldn't be a problem (unless you can only get to the room without any available resources ...). It would kind of be annoying to make it so that the room is no longer free to traverse if Phantoon is alive, but it would be more accurate in general and consistent with how we have handled other rooms.

Reply:
Another option to consider is we could add the enemy requirements consistently but not put "canRiskPermanentLossOfAccess" in the powered off strats? In this way, Map Rando would assume the powered off strats could always be done, making the enemy requirements logically irrelevant (under an assumption that if you can kill Phantoon then you can take care of the enemies, if we believe that assumption is safe?). Even if the enemy kill requirements are effectively ignored by Map Rando, it could still be useful to have them there for other projects, where for example the power might be on from the start without having to kill Phantoon. Seems like it's going to be somewhat awkward no matter which way we go with this. Open to suggestions

Reply:
Attic with enough items to kill phantoon; I'd expect the first few enemies to get a hit in and take some amount of ammo to clear.  Once you're past the initial wave you get some resources back and it should be fine for getting through the rest of the room.  Its hard for missile ammo to be super tight, and if you don't have missiles then you have some good items or a way to dodge.  So I would only want to be sure you can tank a kihunter hit, or dodge.

Reply:
That makes sense. If we want to assume you can get your resources back, we could use a `resourceAvailable` requirement rather than `ammo`; that works for the Missiles at least. We don't really have a version of `enemyDamage` that assumes you have enough energy but doesn't logically remove the energy. There is a farm strat that could apply afterward to fill you back up, but it assumes Wave, Plasma, or Screw. But maybe on low difficulty it's fine to assume you just lose that energy if you don't have the items that would allow farming.

It seems like we may want to create an obstacle to represent enemies being cleared? Otherwise it's going to be a lot of duplication of complex requirements.

Reply:
Trickyjump would be better for this kind of dodging